### ISSUE TITLE
fzf-tmux is no longer updated as new fzf flag for tmux exists

### ISSUE THREAD

**Issue by shadow-absorber** [omerxx/tmux-sessionx#118]
> If you're stuck with an old version of tmux that doesn't support popup, or if you want to open fzf in a regular tmux pane, check out [fzf-tmux](https://github.com/junegunn/fzf/blob/master/bin/fzf-tmux) script.

seems the old fzf-tmux is deprecated and this should be updated to handle the new `--tmux` option instead

**Comment by omerxx** [comment]
Yep! Thanks!
More context: https://github.com/junegunn/fzf?tab=readme-ov-file#--tmux-mode

**Comment by shadow-absorber** [comment]
ah fair.... so any clue how to get the fzf-tmux script on arch linux then in the meantime???? or could you do some fzf version check to handle both older and newer versions???

**Comment by shadow-absorber** [comment]
seems they are on the version 44 on ubuntu 24.04

**Comment by shadow-absorber** [comment]
never mind... seems fzf-tmux is still there in newest version of fzf on arch

**Comment by omerxx** [comment]
@shadow-absorber Yep should be still there, but this issue is super important I'll keep it open

**Comment by blacktop** [comment]
https://github.com/junegunn/fzf/issues/3973#issuecomment-2308395933

**Comment by El-Barto89** [comment]
I cant make it work. Nothing happens when i try to run sessionx with standard keybindings. 
fzf --version → 0.57.0 
TMUX VERSION → 3.4
It is related to fzf-tmux?
